catroid/src/main/java/org/catrobat/catroid/stage/StageResourceHolder.java

```diff
@@ -196,7 +196,8 @@ public void onClick(DialogInterface dialog, int which) {
 										.findViewById(R.id.dialog_terms_of_use_check_box_agree_permanently);
 								PreferenceManager.getDefaultSharedPreferences(stageActivity)
 										.edit()
-										.putBoolean(SETTINGS_PARROT_AR_DRONE_CATROBAT_TERMS_OF_SERVICE_ACCEPTED_PERMANENTLY, checkBox.isChecked());
```

Comment:
no `.commit()` :thinking: how did that even work?

Reply:
I have honestly no idea. In my opinion this should never have worked 🤷